engines/order_management/app/services/order_management/reports/enterprise_fee_summary/scope.rb

```diff
@@ -169,14 +170,38 @@ def include_enterprise_fee_details
           )
         end
 
-        # If for line item - Use data only if spree_line_items.id is present
+        # If for order source
+        #
+        # Includes:
+        # * Source order
+        # * Distributor
+        def include_order_source_details
+          join_scope(
```

Comment:
This method name is a bit ambiguous: https://www.rubydoc.info/github/rails/rails/ActiveRecord%2FReflection%2FAbstractReflection:join_scope

Did you forget to chain the two scopes together?